src/helpers/Libify.js

```diff
@@ -625,6 +671,88 @@ Libify.Operation.setTrustLineFlags = function (opts) {
   });
 };
 
+Libify.Operation.liquidityPoolDeposit = function (opts) {
+  assertNotEmpty(
+    opts.liquidityPoolId,
+    "Liquidity Pool Deposit operation requires Liquidity Pool ID",
+  );
+
+  assertNotEmpty(
+    opts.maxAmountA,
+    "Liquidity Pool Deposit operation requires Max Amount A",
+  );
+
+  assertNotEmpty(
+    opts.maxAmountB,
+    "Liquidity Pool Deposit operation requires Max Amount B",
+  );
+
+  assertNotEmpty(
+    opts.minPrice,
+    "Liquidity Pool Deposit operation requires Min Price",
+  );
+
+  if (opts.minPrice && typeof opts.minPrice === "object") {
+    if (!(opts.minPrice.n && opts.minPrice.d)) {
+      throw new Error(
+        "Liquidity Pool Deposit operation requires both numerator and denominator for Min Price fraction",
+      );
+    }
+  }
+
+  assertNotEmpty(
+    opts.maxPrice,
+    "Liquidity Pool Deposit operation requires Max Price",
+  );
+
+  if (opts.maxPrice && typeof opts.maxPrice === "object") {
```

Comment:
nit, but isn't `opts.[max|min]Price` implied after the assertion?

Reply:
Yup, this is just an extra safety measure.